rms/auth/views.py

```diff
@@ -17,3 +17,18 @@ def login(login_data: LoginRequest):
     response.set_cookie(key="auth_cookie", value=auth_cookie.value, httponly=True, samesite="strict")
 
     return response
+
+
+@router.post("/logout")
+def logout(request: Request):
+    auth_cookie_value = request.cookies.get("auth_cookie")
+    if not auth_cookie_value:
+        raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail="Not Authorized")
+
+    if not invalidate_cookie(auth_cookie_value):
+        raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail="Not Authorized")
+
+    response = Response(content="Logout successful")
```

Comment:
Please make this response return json structured data, so for example
```json
{ "status": "Logouted out successfully" }
```

Reply:
fixed